lib/ansible/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -61,6 +61,11 @@
     default: null
     description: Microsoft Azure subscription ID. defaults to None.
     version_added: "2.5"
+  project:
+    required: false
+    default: null
+    description: Project ID. defaults to None.
```

Comment:
Please mention this is for GCE eg. `description: Google Compute Engine project id`.
Without context, "project id" means nothing...

Reply:
done.